src/Compilers/CSharp/Portable/Symbols/Source/SourceModuleSymbol.cs

```diff
@@ -625,66 +612,12 @@ internal override void AddSynthesizedAttributes(PEModuleBuilder moduleBuilder, r
                 }
             }
-
-            if (UtilizesNullableReferenceTypes)
-            {
-                // PROTOTYPE(NullableReferenceTypes): Support NetModule.
-                if (compilation.Options.OutputKind != OutputKind.NetModule)
-                {
-                    AddSynthesizedAttribute(ref attributes, compilation.TrySynthesizeAttribute(WellKnownMember.System_Runtime_CompilerServices_NullableAttribute__ctor));
-                }
-            }
-        }
-
-        // PROTOTYPE(NullableReferenceTypes): Handle NullableAttribute
-        // consistently with other embedded attributes.
-        internal NamedTypeSymbol GetNullableAttribute(DiagnosticBag diagnostics)
-        {
-            if (!UtilizesNullableReferenceTypes)
-            {
-                return null;
-            }
-            if ((object)_lazyNullableAttribute == null)
-            {
-                Interlocked.CompareExchange(ref _lazyNullableAttribute, GetNullableAttributeAndDiagnostics(_assemblySymbol.DeclaringCompilation), null);
-            }
-            diagnostics.AddRange(_lazyNullableAttribute.Diagnostics);
-            return _lazyNullableAttribute.Attribute;
-        }
-
-        private static AttributeAndDiagnostics GetNullableAttributeAndDiagnostics(CSharpCompilation compilation)
-        {
-            var diagnostics = new DiagnosticBag();
-            var symbol = new SynthesizedEmbeddedAttributeSymbol(
-                AttributeDescription.NullableAttribute,
-                compilation,
-                GetAdditionalNullableAttributeConstructors,
-                diagnostics);
-            return new AttributeAndDiagnostics(symbol, diagnostics);
-        }
-
-        private static ImmutableArray<MethodSymbol> GetAdditionalNullableAttributeConstructors(
-            CSharpCompilation compilation,
-            NamedTypeSymbol containingType,
-            DiagnosticBag diagnostics)
-        {
-            var boolType = compilation.GetSpecialType(SpecialType.System_Boolean);
-            Binder.ReportUseSiteDiagnostics(boolType, diagnostics, Location.None);
-            var boolArray = TypeSymbolWithAnnotations.Create(
-                ArrayTypeSymbol.CreateSZArray(
-                    boolType.ContainingAssembly,
-                    TypeSymbolWithAnnotations.Create(boolType)));
-            return ImmutableArray.Create<MethodSymbol>(
-                new SynthesizedEmbeddedAttributeConstructorSymbol(
-                    containingType,
-                    m => ImmutableArray.Create(SynthesizedParameterSymbol.Create(m, boolArray, 0, RefKind.None))));
         }
 
         internal override bool UtilizesNullableReferenceTypes
         {
             get
             {
-                // PROTOTYPE(NullableReferenceTypes): C#8 projects require System.Attribute.
                 return _assemblySymbol.DeclaringCompilation.IsFeatureEnabled(MessageID.IDS_FeatureStaticNullChecking);
```

Comment:
Is the plan to remove `UtilizesNullableReferenceTypes` altogether? If not, can you clarify what it should mean?

Reply:
There is a `// PROTOTYPE` comment on the base method in `ModuleSymbol` to remove this property.